test/run-tests

```diff
@@ -504,7 +517,18 @@ def get_comment_commands(gh, owner, repo, pullnr):
     return commands
 
 
-def choose_task(gh, repos, images, config):
+def get_status_context(config_name, image_name, ansible_ver_str):
+    suffix = ""
+    if config_name == "linux-system-roles-test":
+        suffix = ""
+    elif config_name == "linux-system-roles-test-staging":
+        suffix = " (staging)"
+    else:
+        suffix = f" ({config_name})"
+    return f"{image_name}/{ansible_ver_str}{suffix}"
+
+
+def choose_task(gh, repos, images, config, ansible_ver_str):
```

Comment:
```suggestion
def choose_task(gh, repos, images, config, ansible_version):
```

Since this is the only ansible version in this context, there is no need to encode the type in the variable name.

Reply:
> Since this is the only ansible version in this context, there is no need to encode the type in the variable name.

Perhaps in this particular context, but earlier in the program execution the code has already set the precedence that the object named by `ansible_version` is indeed the numeric `LooseVersion` compatible primarily numeric (that is, looks like N.M.something where something is primarily numeric or can be alphaN or betaX), so it might be confusing for someone doing a search of this file and finding that `ansible_version` means something different depending on the context.

Perhaps you might examine the code further and find that the code in its current form has such usage where a name is used that has different meanings in different contexts, but does that mean it is important to be consistent at the expense of potential confusion?

Reply:
My confusion came from that `ansible_ver_str` sounds to me like it contains a string representation of the Ansible version, so for example `"2.8.1"` but it actually contains `"ansible-2.8"`, so my new suggestion later in my comments is `ansible_id`.

Reply:
fixed in https://github.com/linux-system-roles/test-harness/pull/87/commits/a1f7e36ec876eb927bb6ddcd25ba619b13dc88c0